chronos/config.nim

```diff
@@ -32,6 +32,17 @@ when (NimMajor, NimMinor) >= (1, 4):
 
     chronosDumpAsync* {.booldefine.}: bool = defined(nimDumpAsync)
       ## Print code generated by {.async.} transformation
+
+    chronosProcShell* {.strdefine.}: string =
+      when defined(windows):
+        "cmd.exe"
+      else:
+        when defined(android):
+          "/system/bin/sh"
+        else:
+          "/bin/sh"
+      ## Default shell binary path.
```

Comment:
good to document how / where this gets used, it's kind of .. unusual .. to have to have a shell

Reply:
done.